modules/standard/FileSystem.chpl

```diff
@@ -21,12 +21,12 @@
 
    The FileSystem module focuses on file and directory properties and
    operations.  It does not cover every interaction involving a file -
-   for instance, path specific operations live in the :mod:`Path` module, while
-   opening, writing to, or reading from a file appropriately live in the
+   for instance, path-specific operations live in the :mod:`Path` module, while
+   opening, writing to, or reading from a file live in the
    :mod:`IO` module.  Rather, it covers cases where the user would prefer a file
-   or directory handled wholesale and/or with minimal interaction.  The
-   :proc:`copy` functionality belongs here, as does :proc:`chmod`,
-   :proc:`mkdir`, and :proc:`remove`, for instance.  Also included are
+   or directory handled wholesale and/or with minimal interaction.  For example,
+   this module contains file :proc:`copy` functionality, as well as :proc:`chmod`,
```

Comment:
There are some categories that occurred to me: for instance, all the functions which would use and deal with a file's mode (and the constants associated with it) could go in a single category, while all the locale specific methods would fit into another category.  Functions that tell you information about a file or directory, such as isFile, exists, etc., could be a third, and those that create or remove files (like mkdir, remove, symlink, copy) could be a fourth.